src/vorta/__main__.py

```diff
@@ -5,13 +5,33 @@
 import peewee
 from vorta._version import __version__
 from vorta.config import SETTINGS_DIR
-from vorta.log import init_logger
+from vorta.log import init_logger, logger
 from vorta.models import init_db
 from vorta.updater import get_updater
 from vorta.utils import parse_args
 
 
 def main():
+    def exception_handler(type, value, tb):
+        from traceback import format_exception
+        from PyQt5.QtWidgets import QMessageBox
+        logger.critical("Uncaught exception, file a report at https://github.com/borgbase/vorta/issues/new",
+                        exc_info=(type, value, tb))
+        if app and app.main_window:
+            full_exception = ''.join(format_exception(type, value, tb))
+            try:
+                QMessageBox.critical(app.main_window,
+                                     app.main_window.tr("Fatal Error"),
+                                     app.main_window.tr(
+                                         "Uncaught exception, please file a report with this text at\n"
+                                         "https://github.com/borgbase/vorta/issues/new\n") + full_exception)
+            except RuntimeError:
+                # Window is closed, only log is available
```

Comment:
Not possible to show a messagebox without main window?

Reply:
It would have the strings untranslated, since the translation comes from main window.

Reply:
Probably not an issue and new translations are added over time.

Reply:
I mean that the translation is impossible, since it is inherited from the crashed application

Reply:
Then we may have a message window title that can be translated and the details on the exception below are always as-is untranslated, correct?

Reply:
Found a way to do it, now it doesn't show translations when there is no `app`